test/src/specs/image/image_attribute_test.dart

```diff
@@ -52,18 +52,26 @@ void main() {
       const attribute = ImageSpecAttribute(
         width: 100,
         height: 200,
-        color: ColorDto(Colors.red),
+        color: ColorDto(Colors.black),
         repeat: ImageRepeat.repeat,
         fit: BoxFit.cover,
+        alignment: Alignment.bottomCenter,
+        centerSlice: Rect.zero,
+        filterQuality: FilterQuality.low,
+        colorBlendMode: BlendMode.srcOver,
       );
       final props = attribute.props;
 
-      expect(props.length, 5);
+      expect(props.length, 9);
```

Comment:
I think we don;t need to do these props check, as they are hard to maintain, and I think some `dcm` fixes might reorder them. The most important thing is to check the values, and also check `equality` on the tests.